edbterraform/lib.py

```diff
@@ -266,7 +270,7 @@ def new_project_main():
         '--cloud-service-provider', '-c',
         metavar='CLOUD_SERVICE_PROVIDER',
         dest='csp',
-        choices=['aws', 'gcloud', 'azure'],
+        choices=['aws', 'gcloud'],
```

Comment:
This file is undoing https://github.com/EnterpriseDB/edb-terraform/pull/16 merged changes.
`azure` is supported

Reply:
Updated